VMEncryption/main/handle.py

```diff
@@ -266,26 +289,13 @@ def update_encryption_settings():
             # store new passphrase and overwrite old encryption key file
             bek_util.store_bek_passphrase(encryption_config, extension_parameter.passphrase)
 
-            # post new encryption settings via wire server protocol
-            settings = EncryptionSettingsUtil(logger)
-            new_protector_name = settings.get_new_protector_name()
-            settings.create_protector_file(new_protector_name)
-            data = settings.get_settings_data(
-                protector_name=new_protector_name,
-                kv_url=extension_parameter.KeyVaultURL,
-                kv_id=extension_parameter.KeyVaultResourceId,
-                kek_url=extension_parameter.KeyEncryptionKeyURL,
-                kek_kv_id=extension_parameter.KekVaultResourceId,
-                kek_algorithm=extension_parameter.KeyEncryptionAlgorithm)
-            settings.write_settings_file(data)
-            settings.post_to_wireserver()
-            settings.remove_protector_file(new_protector_name)
+            stamp_disks_with_settings(new_device_items_about_to_get_encrypted=[], encryption_config=encryption_config)
 
             # commit local encryption config
-            encryption_config.passphrase_file_name = extension_parameter.DiskEncryptionKeyFileName
-            encryption_config.secret_id = new_protector_name
-            encryption_config.secret_seq_num = hutil.get_current_seq()
-            encryption_config.commit()
+            # encryption_config.passphrase_file_name = extension_parameter.DiskEncryptionKeyFileName
```

Comment:
consider deleting these lines (295-298), they're still in version control if anyone needs to look back